cart/src/test/java/com/yas/cart/service/CartServiceTest.java

```diff
@@ -0,0 +1,62 @@
+package com.yas.cart.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.anyList;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.yas.cart.model.Cart;
+import com.yas.cart.repository.CartItemRepository;
+import com.yas.cart.repository.CartRepository;
+import com.yas.cart.viewmodel.CartItemVm;
+import com.yas.commonlibrary.exception.BadRequestException;
+import com.yas.commonlibrary.exception.NotFoundException;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CartServiceTest {
```

Comment:
Why there are no happy cases in this test class?

Reply:
This cart test class is designed solely to test the functionality of the common library. Since the cart module already has high code coverage, I’ve only added a few cases to trigger the pipeline